.README/rules/require-example.md

```diff
@@ -57,7 +62,7 @@ report a missing example description after this is added.
 |Context|`ArrowFunctionExpression`, `FunctionDeclaration`, `FunctionExpression`; others when `contexts` option enabled|
 |Tags|`example`|
 |Recommended|false|
-|Options|`exemptedBy`, `exemptNoArguments`, `avoidExampleOnConstructors`, `contexts`|
+|Options|`exemptedBy`, `exemptNoArguments`, `contexts`, `checkConstructors`, `checkGetters`, `checkSetters`, `enableFixer`|
```

Comment:
This removes a deprecated option and adds the options that were missing from this table.

Reply:
Oh, good find. Doing a search, I also see `avoidExampleOnConstructors` is within the main README. Would you mind updating that too?

Reply:
Done!